spec/javascript/support/dom.js

```diff
@@ -94,13 +94,13 @@ export function createDOM() {
  */
 export function useCleanDOM(dom) {
   beforeEach(() => {
-    while (document.body.firstChild) {
-      document.body.removeChild(document.body.firstChild);
+    for (const element of [document.head, document.body]) {
+      while (element.firstChild) {
+        element.firstChild.remove();
```

Comment:
`Element#remove` wasn't supported in IE11 which is why you see `parent.removeChild(child)` in a lot of sample snippets. We shouldn't need to worry about that compatibility in the context of specs, since JSDOM supports `Element#remove`.